requirements/prod.in

```diff
@@ -1,6 +1,6 @@
 ansi2html
 boto3
-cumulusci==3.35.0.dev0
+cumulusci==3.36.0
```

Comment:
We don't need/want the version here normally. I added it temporarily to get the pre-release.